apps/trading/pages/index.page.tsx

```diff
@@ -37,14 +40,15 @@ export function Index() {
       // If a default market is found, go to it with the landing dialog open
       if (marketId) {
         setLandingDialog(true);
+        setMarketId(marketId);
```

Comment:
This is because the trading link in the nav should navigate to the last used market (see logic in the trading nav). I don't love this UI pattern but its needed for it to work